components/metricsView/cards/UpTimeNode.component.tsx

```diff
@@ -18,20 +16,8 @@ type UpTimeProps = {
   show: (show: boolean, message: string) => void;
 };
 
-interface AxisCustomizedProps extends AxisProps {
-  orient?: string;
-  color?: string;
-}
-interface ResponsiveLineProps extends LineSvgProps {
-  axisBottom: AxisCustomizedProps;
-  axisLeft: AxisCustomizedProps;
-}
-function ResponsiveLine(props: ResponsiveLineProps) {
-  return <Line {...props} />;
-}
-
 export default function UpTimeNode({ nodeInfo, metrics, show }: UpTimeProps) {
-  let lineChartData: Array<Serie> = [];
+  let lineChartData: any = [];
```

Comment:
why we are removing these? can be helpfult to know what data are inside the array no?

Reply:
Yes we can have the `Serie` but the code before that is not needed.

Reply:
Can you explain why? I think there type declaration are connected!

I would like to introduce the type, because any is bad

Reply:
Yup changed back to `Serie`